src/UI/TransactionsToastList/index.tsx

```diff
@@ -141,8 +142,14 @@ export const TransactionsToastList = ({
     ]
   );
 
-  const customToastsList = customToasts.map(
-    ({ toastId, type, duration, message = '' }: CustomToastType) => (
+  const customToastsList = customToasts
+    // Avoid rendering transaction status toast.
+    // This is rendered individually in case of canceled/error transaction
+    .filter(
+      (customToast: CustomToastType) =>
```

Comment:
This logic is a bit contrived. We have custom toasts that can be opened and closed, and we use those for transaction toast statuses, but we don't render them together. Should we maybe add a separate slice/entry for customToast statuses, so we don't mix these up, since it's not the same logic?

Reply:
I tried to reuse useCustomHook as many as possible, but I think is better to create separate logic for this toast